sa/_db-next/migrations/20191108171706_RemoveOldAuthzTables.sql

```diff
@@ -0,0 +1,41 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+DROP TABLE authz;
```

Comment:
It looks like you also need to update `sa/database_test.go`'s `TestStrictness` unit test for this branch. It happens to use the `authz` table in a one-off query:

https://github.com/letsencrypt/boulder/blob/e392b28e76701de2b5abf6a00b6eb317b999eec4/sa/database_test.go#L68-L70

Having that table gone when this migration is applied is causing CI failure:

```
--- FAIL: TestStrictness (0.00s)
    database_test.go:75: Got wrong type of error: Error 1142: INSERT command denied to user 'sa'@'10.77.77.77' for table 'authz'
FAIL
FAIL	github.com/letsencrypt/boulder/sa	7.644s
```
```